Scripts/Python/xOpeningSequence.py

```diff
@@ -75,7 +75,7 @@
 
 gIntroMovie = None
 kAtrusIntroMovie = "avi/AtrusIntro.webm"
-kYeeshaIntroMovie = "avi/URULiveIntro.webm"
+kIntroMovie = "avi/NewPlayerIntro.webm"
```

Comment:
Having variables named `gIntroMovie` and `kIntroMovie` (particularly when `kIntroMovie` is not entirely a constant) seems prone to confusion...

Maybe something like `gMovieFilePath` makes more sense as the variable, and have `kAtrusIntroMovie` and `kReltoIntroMovie` as string constants that get assigned to that filepath variable (along with the vault override)?

Reply:
Done